manifests/vhost.pp

```diff
@@ -338,23 +338,24 @@
   }
 
   if $ip {
+    $_ip = any2array($ip)
     if $port {
-      $listen_addr_port = "${ip}:${port}"
-      $nvh_addr_port = "${ip}:${port}"
+      $listen_addr_port = parseyaml(inline_template('<% res="" %><% @_ip.each { |a_ip| res = res + "#{a_ip}:#{@port}," } %><%= res.split(",").to_yaml -%>'))
```

Comment:
this hole line can be broken up into one call to [suffix()](https://github.com/puppetlabs/puppetlabs-stdlib#suffix):
```puppet
$listen_addr_port = suffix($_ip, $port)
```